### ISSUE TITLE
Can't send XTZ from my TezBox wallet

### ISSUE THREAD

**Issue by jdjennings96** [tezbox/desktop-wallet#20]
Hello - I am trying to send some XTZ out of of my TezBox wallet and I keep getting an error message that reads "Operation Failed - Please check your inputs".  I have entered the correct recipient address    Here are the settings:
Fee:  set to default:  Low - 1450 micro XTZ
Parameters: set to default:  nothing
Gas Limit: set to default 10300    
Storage Limit:  set to default - 300 
Can you advise on why i'm getting this error message?   Thx

**Comment by tracdinh** [comment]
i have same issue

**Comment by jdjennings96** [comment]
Any idea how to resolve this?

**Comment by tracdinh** [comment]
use galleon restore wallet or wait til tezbox update new version. 
https://galleon-wallet.tech

**Comment by jdjennings96** [comment]
Ok...Will try this.  Thanks for the advice.   J

**Comment by jdjennings96** [comment]
Will I be using Galleon to repair my TezBox wallet?  Or do I need to create a Galleon wallet and transfer the XTZ?  Not clear on which of the videos to follow.  Thx.

**Comment by tracdinh** [comment]
use galleon restore with your seed words. all your xtz will in there and now u can tranfer to anywhere u want

**Comment by TezboxAdmin** [comment]
i found a fix to this isusse go to Tezbox wallet with Chrome or Firefox browser:

[https://wallet.tezbox.com](https://www.tezbox.io/#!/restore)

and restore your wallet then go to setting and find better node

**Comment by shaun19826** [comment]
Hello, have you been able to resolve your issue ?

**Comment by jdjennings96** [comment]
No.  I am still having difficulties.  I think I am still doing something wrong.  I will try it again tomorrow night.  Is there someone I could call to discuss the fix if I still can’t get it resolved this week?

**Comment by shaun19826** [comment]
You can send me a message on telegram, my contact is +447432185658. This number is for my telegram only.
I'll be happy to help.
Cheers mate

**Comment by jdjennings96** [comment]
Ok. Thx!  I don’t have time to get to it as anticipated tonight but I will try again by the weekend and contact you if I still can’t move the funds or reload the wallet.  Really appreciate the help.

**Comment by shaun19826** [comment]
Okay that’s fine.
Weekend it is then.
I’ll be expecting your message.
Cheers.

**Comment by shaun19826** [comment]
Hello,
I had the same issue too.
Tezbox is poorly maintained. It is highly recommend to use another wallet. Your address is on blockchain and can be accessed by many wallets, easiest one of them is the Galleon wallet.

I'll recommend you migrate to the galleon wallet.
All you need to do is to import your fundraiser PDF doc (if you participated) OR Restore you wallet using your private key (most preferred and advisable) or 12 word backup phrase on the galleon web wallet on www.galleon.network

You’ll have access to your funds upon a successful restoration
cheers mate.

> Ok. Thx! I don’t have time to get to it as anticipated tonight but I will try again by the weekend and contact you if I still can’t move the funds or reload the wallet. Really appreciate the help.
> […](#)

**Comment by shaun19826** [comment]
Hello @jdjennings96 ,
Have you been able to resolve the issue yet ?

**Comment by shaun19826** [comment]
Hello,
I had the same issue too.
Tezbox is poorly maintained. It is highly recommend to use another wallet. Your address is on blockchain and can be accessed by many wallets, easiest one of them is the Galleon wallet.

I'll recommend you migrate to the galleon wallet.
All you need to do is to import your fundraiser PDF doc (if you participated) OR Restore your wallet using your private key (most preferred and advisable) or 12 word backup phrase on the galleon web wallet on www.galleon.network

You’ll have access to your funds upon a successful restoration
Cheers mate.

**Comment by jdjennings96** [comment]
Hi Shaun - Ok I finally had time to look at this today.  I actually did open a Galleon wallet and did a restore in March.  The XTZ is there and it looks like I was receiving baking rewards while it was in my Tezbox wallet.  I thought I assigned a new baker when I created the Galleon wallet but maybe not since I don’t see any reward txs in the wallet.  Also after creating the Galleon wallet I tried to move some of the XTZ to my Atomic wallet and I could not get the transactions to go thru.  My XTZ is a small portion of my portfolio so I haven’t been paying a lot of attention to it.  But it does seem like an asset that is difficult to use.  I have similar troubles with my EOS.  What do you recommend for an XTZ block explorer?  And for a guide on how to use the Galleon wallet?  I obviously need to get a baker assigned and learn how to move XTZ in and out.  Thx!  

Also, I am in the US.  Where are you located?  Sounds like Australia or NZ from your messages.  Really appreciate your assistance.  No one around me is into digital assets so it gets frustrating when I run into something that’s not working.  Best,   J

**Comment by shaun19826** [comment]
Hello @jdjennings96 ,  To get your baker assigned, restore your wallet on the galleon web wallet on www.galleon.network which gives you full access to all features including list of all bakers with high payout options and low fee charges.
It’s simpler to use the galleon web wallet as it entails more details about the bakers, when using the galleon web wallet you’re allowed to choose from a list of bakers  of which their payout options and fees will be discussed.

You can do that and get back to me on whatever your outcome is

**Comment by shaun19826** [comment]
Hello @jdjennings96 , To get your baker assigned, restore your wallet on the galleon web wallet on www.galleon.network which gives you full access to all features including list of all bakers with high payout options and low fee charges.
It’s simpler to use the galleon web wallet as it entails more details about the bakers, when using the galleon web wallet you’re allowed to choose from a list of bakers of which their payout options and fees will be discussed.

You can do that and get back to me on whatever your outcome is.
Cheers

**Comment by shaun19826** [comment]
Hello @jdjennings96 , To get your baker assigned, restore your wallet on the galleon web wallet on www.galleon.network which gives you full access to all features including list of all bakers with high payout options and low fee charges.
It’s simpler to use the galleon web wallet as it entails more details about the bakers, when using the galleon web wallet you’re allowed to choose from a list of bakers of which their payout options and fees will be discussed.

You can do that and get back to me on whatever your outcome is.
Cheers

**Comment by shaun19826** [comment]
@jdjennings96 Hi mate

**Comment by jdjennings96** [comment]
Hi Shaun - I have not had time to go thru the Galleon website you advised.  I plan to do that this week and will let you know if I have any troubles using the service to set up a baker.  Appreciate all the advice.   J

**Comment by shaun19826** [comment]
Okay no problem.
Ensure you’re doing that via the Galleon web wallet on www.galleon.network

You can message for more information
Cheers

**Comment by shaun19826** [comment]
Hello @jdjennings96

**Comment by shaun19826** [comment]
@jdjennings96 Hello mate !
What's the update on the steps I gave you earlier in restoring your wallet ?

**Comment by jdjennings96** [comment]
Hi Shaun - I tried the www.galleon.network website.  I selected the Restore
From Backup tab and entered my seed phrase, but nothing happened.  I tried
it a few times with the same results. So I opened my Galleon Wallet (I am
using version 0.9.4b) and reviewed it again.  My main problem was not being
able to move the XTZ but for now I'm just going to leave it there and
update the baker to start getting baking rewards going again.  I used the
Baking Bad website to select a baker.  I do have a further question though
for when I do decide to try moving the tokens again:  Do you have to move
your XTZ from the Delegation Contract "kt" address to the manager "tz"
address before you can move it out of the Galleon wallet?  Thanks.     J

**Comment by jdjennings96** [comment]
Hi Shaun - I am not sure why this is so difficult for me but I could really
use your help again.  I staked my XTZ in my Galleon wallet on July 26 and
now it does not show in my wallet and I don't seem to be receiving any
rewards that I can see.  Here's the detail that I see in the wallet:

Transaction on July 26, 2020
Transfer from my wallet (tz1W5wqtNAJcUwurhuovWtUfmGz6nrVyDefb) to the
following wallet (tz1c8Y3x2AHdAZ3TXdDYKZ7PjZSUWx69cNKY) of 1793.0  XTZ
On this date I selected "Tezgate" (address below as the baker so I assumed
the funds were being transferred to the Tezgate pool):
tz1W5VkdB5s7ENMESVBtwyt9kyvLqPcUczRT

Are you able to help me determine where these tokens are and whether they
are correctly staked?

Thanks so much again for your advice.   J

**Comment by shaun-19826** [comment]
@jdjennings96
 Hello,

Apologies for the inconvenience you're having. I had the same issue too in August.
Have you tried adding funds to your Tz1 address tz1W5wqtNAJcUwurhuovWtUfmGz6nrVyDefb ?

If you haven't, i'll suggest you add a minimum of 300xtz to get started. Upon add the new funds, your previous and newly added funds shows up.

You can give that a try and get back to me when you've done that.
Cheers mate.

**Comment by jdjennings96** [comment]
Hi Shaun - I added the 300 XTZ but my delegated tokens still don't show.
Any other suggestions?  It looks like they were sent to  tz1
c8Y3x2AHdAZ3TXdDYKZ7PjZSUWx69cNKY when I opened the delegation contract.
Can I contact this baker and ask them if the funds are staked and earning
rewards?  I don't understand why I don't see the staked tokens and baking
transactions.   J

**Comment by shaun-19826** [comment]
Hello @jdjennings96, do you have any other Tezos account asides from this current one ?

**Comment by jdjennings96** [comment]
Ok so in the Galleon Wallet I have the following public addresses:
Manager Address: tz1W5wqtNAJcUwurhuovWtUfmGz6nrVyDefb
Delegation Contract 1 Address:  KT1MZ1smQCMGf1QxVs9ThfFhHvgxkwRjQjzo

I also have the TezBox Wallet (which looks like it has the same addresses):
Main Address: tz1W5wqtNAJcUwurhuovWtUfmGz6nrVyDefb
Account 1 Address:  KT1MZ1smQCMGf1QxVs9ThfFhHvgxkwRjQjzo

I also have some XTZ on Atomic Wallet:
Atomic: tz1igHigUQ5JDDSCYc7ZswVLkYb51DWU6N2M

Hope this helps to sort out the missing delegated tokens...I really
appreciate your assistance as I don't know that I could triage this on my
own.    J

**Comment by shaun-19826** [comment]
@jdjennings96 Okay. Thats good. I've trying to know if you had another wallet. Thats perfect, all will be sorted out now in few minutes. Since you have an atomic wallet, restore your atomic wallet account using your 12 word backup phrase on the galleon web wallet on www.galleon.network. On opening the galleon web wallet website, you'll be restoring from backup.

Let me know when you've done that so that your galleon account can be linked with that of your atomic account, thereby showing your delegated funds on the atomic wallet. Thus, solving all issues.

**Comment by shaun-19826** [comment]
@jdjennings96 Have you been able to follow the procedures I listed above ?

**Comment by jdjennings96** [comment]
No, I haven't. I just saw your email.  I will try that, but I'm curious as
to how that would show me the missing tokens.  I delegated them to another
baker so how would linking my Atomic wallet to my Galleon wallet get me
access to the missing delegated tokens?  Also, I am staking the XTZ in the
Atomic wallet so that won't cause a problem if I link the Galleon wallet,
correct?   Thanks.     J

**Comment by shaun-19826** [comment]
@jdjennings96 Of course not. it won't cause a problem with your atomic wallet. Due to the fact that you haven't used your galleon wallet in a while, it needs an existing wallet which is currently in use. that's why I said you should link your atomic wallet with the procedures I left above. You can do that right away so I can help further if you still don't get access to the delegated Tezos.

**Comment by jdjennings96** [comment]
Got it. Ok, I will try doing the link now.

**Comment by shaun-19826** [comment]
@jdjennings96 Okay, I'll be here in case of any errors

**Comment by jdjennings96** [comment]
The Galleon Wallet Restore is asking for a 15 word seed phrase and my
Atomic wallet seed phrase is only 12 word.  Will a 12 word phrase work?
And do I use spaces or commas between the words?

**Comment by shaun-19826** [comment]
Yes, the 12 word phrase will work. Also use spaces to separate the words.

**Comment by jdjennings96** [comment]
Will do. Thx

**Comment by shaun-19826** [comment]
@jdjennings96 Okay, I'll be here in case of any errors

**Comment by jdjennings96** [comment]
Ok. I did the restore but it didn't seem to work.  After I entered the seed
phrase and clicked restore it just opens up the Import Fundraiser Wallet
window.  I closed the Galleon Wallet and reopened it thinking maybe I had
to exit and reopen to see the added account but the atomic wallet account
still does not show up.  Thoughts?
